packages/@aws-cdk/aws-apigatewayv2/lib/http/integration.ts

```diff
@@ -29,9 +30,83 @@ export enum HttpIntegrationType {
   LAMBDA_PROXY = 'AWS_PROXY',
   /**
    * Integration type is an HTTP proxy
-   * @see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html
+   * @see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-http.html
    */
   HTTP_PROXY = 'HTTP_PROXY',
+  /**
+   * Integration type is an AWS proxy
+   * @see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services.html
+   */
+  AWS_PROXY = 'AWS_PROXY',
```

Comment:
Having different enum members with the same value causes problems for jsii when generating code in other languages.

Reply:
OK, what do you suggest?

Reply:
Given that this is an experimental module, we can introduce a breaking change and remove the `LAMBDA_PROXY` member.